components/brave_wallet_ui/page/screens/swap/components/swap/max_slippage/max_slippage.style.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+import styled from 'styled-components'
+import * as leo from '@brave/leo/tokens/css/variables'
+import Input from '@brave/leo/react/input'
+import Icon from '@brave/leo/react/icon'
+
+// Shared Styles
+import {
+  WalletButton //
+} from '../../../../../../components/shared/style'
+
+export const OptionButton = styled(WalletButton)<{
+  isSelected: boolean
+}>`
+  --radio-icon-color: ${(p) =>
```

Comment:
can we move this to the RadioIcon for clarity?

Reply:
Moved!